logsmanagement/README.md

```diff
@@ -0,0 +1,519 @@
+
+To configure Netdata's logs management engine properly, please make sure you are aware of the following points:
+
+* One collection cycle (at max) occurs per `update every` interval (in seconds) and any log records collected in a collection cycle are grouped and compressed together. As a result, a longer `update every` interval will reduce memory and disk space requirements.
+* When collected logs contain parsable timestamps, these will be used to display metrics from parsed logs at the correct time in each chart, even if collection of said logs takes place *much* later than the time they were produced. How much later? Up to a configurable value of `update timeout` seconds. This mechanism ensures correct  parsing and querying of delayed logs that contain parsable timestamps (such as streamed inputs or buffered logs sources that write logs in batches), but the respective charts may lag behind some seconds up to that timeout. If no parsable timestamp is found, the collection timestamp will be used instead.
+
+<a name="collector-types"/>
+
+### Types of available collectors
+
+</a>
+
+The following log collectors are supported at the moment. The table will be updated as more collectors are added:
+|  Collector    	| Log type      		| Description  |
+| ------------  	| ------------  		| ------------ |
+| kernel logs (kmsg)| `flb_kmsg`    		| Collection of new kernel ring buffer logs.|
+| systemd       	| `flb_systemd` 		| Collection of journald logs.|
+| docker events 	| `flb_docker_events` 	| Collection of docker events logs, similar to executing the `docker events` command.|
+| web log       	| `flb_web_log` 		| Collection of Apache or Nginx access logs.|
+| generic (tail)	| `flb_generic` 		| Collection of new logs from files by "tailing" them.|
+| syslog socket   	| `flb_syslog`  		| Collection of RFC-3164 syslog logs by creating listening sockets.|
+| serial        	| `flb_serial`  		| Collection of logs from a serial interface.|
+
+<a name="package-requirements"/>
+
+## Package Requirements
+
+</a>
+
+Netdata logs management introduces minimal additional package dependencies and those are actually [Fluent Bit dependencies](https://docs.fluentbit.io/manual/installation/requirements). The only extra build-time dependencies are:
+- `flex` 
+- `bison` 
+- `musl-fts-dev` (Alpine Linux only)
+
+However, there may be some exceptions to this rule as more collectors are added to the logs management engine, so if a specific collector is disabled due to missing dependencies, please refer to this section or check [Troubleshooting](#troubleshooting).
+
+<a name="requirements-systemd"/>
+
+### Systemd
+
+</a>
+
+If systemd development libraries are missing at build time, the systemd log collector will not be available. This can be fixed by installing the missing libraries prior to building the agent:
+
+Debian / Ubuntu:
+
+```
+apt install libsystemd-dev
+```
+Centos / Fedora:
```

Comment:
You wrote `Centos/Fedora` instead `RH family`, do we have different packages for `Alma` and `Rocky`?

Reply:
No, we don't have different packages, that's the package for every RHEL derivative as it seems. I've update this part and also added the equivalent openSUSE command.